distuv/beta.go

```diff
@@ -88,6 +109,17 @@ func (b Beta) StdDev() float64 {
 	return math.Sqrt(b.Variance())
 }
 
+// Survival returns the survival function (complementary CDF) at x.
+func (b Beta) Survival(x float64) float64 {
+	if x <= 0 {
```

Comment:
could perhaps be rewritten as a switch? (I personally find it more readable. but YMMV)

```go
switch {
case x <= 0:
    return 1
case x >= 1:
    return 0
}
```